### PR TITLE
Feature flags admin

### PR DESCRIPTION
**Pull request by nickstenning**
Currently, the only way to change the values of feature flags is through the database. This is non-ideal.

This PR adds a very basic web interface for modifying feature flag configuration.

### CODE REVIEW

File: h/admin.py [hunk 1]
```diff
@@ -16,6 +17,31 @@ def index(_):
...
+                  permission='admin')
+def features_index(_):
+    return {"features": models.Feature.all()}
+
+
+@view.view_config(route_name='admin_features',
+                  request_method='POST',
+                  permission='admin')
+def features_save(request):
+    session.check_csrf_token(request)
```
Comment (tilgovi) [inline h/admin.py]: You can just use the `check_csrf=True` predicate on the view config. Not sure you need to test it at that point.

Comment (nickstenning) [inline h/admin.py]: You can, but that results in a 404 for the POST, which is a bit weird.

**Comment by nickstenning** [comment]
screen shot 2015-08-07 at 18 23 26

**Comment by lenazun** [comment]
nice 🎌

File: h/admin.py (hunk 1 above)
Comment (tilgovi) [inline h/admin.py]: Ah, fair enough. That's unfortunate.

**Pull request merged**